src/autolabel/configs/config.py

```diff
@@ -135,7 +135,7 @@ def input_columns(self) -> List[str]:
 
     def explanation_column(self) -> str:
         """Returns the name of the column containing an explanation as to why the data is labeled a certain way"""
-        return self._dataset_config.get(self.EXPLANATION_COLUMN_KEY, None)
+        return self._dataset_config.get(self.EXPLANATION_COLUMN_KEY, "explanation")
```

Comment:
constant

Reply:
Will make a constant. Thanks!

Reply:
Actually, do we have a separate file for storing constants for default config values? I think this might be it.